portfolio.html

```diff
@@ -17,21 +17,20 @@
 
 <!--------Navbar-------->
 
-    <!--------Video-------->
 
     <div class="video_container">
         <video src="video/99544-1080.mp4" type="video/mp4" loop autoplay></video>
```

Comment:
i couldn't see your video, just a still background image

Reply:
It's not really a video, more like an animated picture. Maybe it wasn't clear enough.